setup/setup-ci/security-scanner/tests/README.md

```diff
@@ -0,0 +1,81 @@
+# Unit tests
+
+To get started, create a conda environment:
+```bash
+conda create --yes --name test-amlsecscan python=3.8
+conda activate test-amlsecscan
+pip install requests~=2.28 pytest~=7.1
+```
+
+Then run the unit tests:
+```bash
+pytest
+```
+
+# Integration tests
+
+## Test on an existing Compute Instance
+
+Open a terminal on the Compute Instance (with `root` user enabled) and run:
+```bash
+sudo ./amlsecscan.py install
+./amlsecscan.py heartbeat
+sudo ./amlsecscan.py scan all
+```
+
+Check Log Analytics for hearbeats and scan status in table `AmlSecurityComputeHealth_CL` and assessment results in `AmlSecurityComputeAssessments_CL`.
+
+## Test on a new Compute Instance
+
+### Using Azure ML Studio
+
+Open the [Compute Instance list](https://ml.azure.com/compute/list) in [Azure ML Studio](https://ml.azure.com), then click the `+ New` button.
+In the pop-up, select a small `Standard_F2s_v2` as machine size, then click `Next: Advanced Settings`.
+Set a schedule to shutdown daily. Then toggle `Provision with setup script`, select `Local file`, and pick `amlsecscan.sh`.
+
+After the Compute Instance is successfully created, check the Compute Instance logs under `Setup > stdout`. There should be no error.
+
+Then check Log Analytics for hearbeats using query `AmlSecurityComputeHealth_CL | top 100 by TimeGenerated desc`.
```

Comment:
Link to how to generically work with log analytics?

Reply:
Added link